eav/models.py

```diff
@@ -595,6 +603,16 @@ def get_value_by_attribute(self, attribute):
         '''
         return self.get_values().get(attribute=attribute)
 
+    def get_object_attributes(self):
```

Comment:
This seems a bit *iffy* to me. Do you guys have any better ideas on how to do this?

Reply:
@IwoHerka I'd go without copying of the dict for:

`return set(attrs.keys()) - set(['instance', 'ct'])`

or, if not using this method anywhere else, just integrate it with validation method.

Reply:
Better yet, set(attrs.keys() - ['instance', 'ct'])

Reply:
The last one is cool. :)

Okay, so I take it there are no objections in using `__dict__`. I was worried about that.

Reply:
Fine with me.

Reply:
`('instance', 'ct')` would be nicer imo, you do not need mutable data structures, but yeah, the solution above is a good one.

Do you think to make this tuple a module-level variable, for example RESERVED_NAMES?

Reply:
@dragonee No, I don't `RESERVED_NAMES` is a good idea.

`instance` and `ct` are Entities attributes, there is no point to parameterize their names
in validation. In this case we just _refer_ to those attributes with strings.